satpy/scene.py

```diff
@@ -29,13 +29,14 @@
 from pyresample.geometry import AreaDefinition, BaseDefinition, CoordinateDefinition, SwathDefinition
 from xarray import DataArray
 
+from satpy.area_utils import get_area_def
 from satpy.composites import IncompatibleAreas
 from satpy.composites.config_loader import load_compositor_configs_for_sensors
 from satpy.dataset import DataID, DataQuery, DatasetDict, combine_metadata, dataset_walker, replace_anc
 from satpy.dependency_tree import DependencyTree
 from satpy.node import CompositorNode, MissingDependencies, ReaderNode
 from satpy.readers.core.loading import load_readers
-from satpy.resample import get_area_def, prepare_resampler, resample_dataset
+from satpy.resample.base import prepare_resampler, resample_dataset
```

Comment:
Since pyresample is a hard requirement this probably doesn't matter, but I'm wondering if this import should be in-function to avoid import Resampler classes. :man_shrugging: Eh that will probably go away naturally when more things are moved to pyresample.

Reply:
... which doesn't really change anything because the functions a imported anyway and thus import from Pyresample :man_facepalming:

Reply:
Moved the imports in https://github.com/pytroll/satpy/pull/3124/commits/7ff4f6df8d4dcc81db0aefecd5dd57c105712305